sheetwhat/checks/check_funcs.py

```diff
@@ -24,12 +24,15 @@ def check_range(state, field, field_msg, missing_msg=None):
     )
 
 
-def has_code(state, pattern, fixed=False, incorrect_msg=None, normalize=lambda x: x):
+def has_code(state, pattern, fixed=False, index=0, incorrect_msg=None, normalize=lambda x: x):
     child = check_range(state, field="formulas", field_msg="formula")
 
     def match(on_text):
         if fixed:
-            return normalize(pattern) in str(on_text)
+            if index == 0:
+                return normalize(pattern) in str(on_text)
+            else:
+                return len(re.findall(normalize(pattern), str(on_text))) >= (index + 1)
```

Comment:
Does this work if the pattern contains regex characters (which is one of the use cases for `fixed=True`)?